backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -408,6 +408,32 @@ func (ctx *AppContext) UpdateUserDataFromMySQL() error {
 	return nil
 }
 
+// MyMasterPubKeys returns a map of master pubkeys to their account
+func (ctx *AppContext) MyMasterPubKeys() (map[string]string, error) {
+	acctMap, err := ctx.WalletConnection.ListAccounts()
+	if err != nil {
+		log.Errorf("MyMasterPubKeys: ListAccounts rpc failed: %v", err)
+		return nil, err
+	}
+	keyMap := make(map[string]string)
+	for acct := range acctMap {
+		// skip imported
+		if acct == "imported" {
+			continue
+		}
+		key, err := ctx.WalletConnection.GetMasterPubkey(acct, ctx.Params)
+		if err != nil {
+			log.Errorf("MyMasterPubKeys: GetMasterPubKey rpc failed: %v", err)
+			continue
+		}
+		keyMap[acct] = key.String()
+	}
+	if len(keyMap) == 0 {
```

Comment:
When looking at the dcrd code, did you confirm whether we need to keep this check or not?

Reply:
This should never happen.  Since we are going to stop on any error I think it is safe to take this out.